diff.py

```diff
@@ -2153,8 +2159,14 @@ def do_diff(lines1: List[Line], lines2: List[Line], config: Config) -> Diff:
                     btset.add(bt)
                     sc(str(bt))
 
+    if len(lines1) > len(lines2) and is_nops(lines1[len(lines2):]):
+        lines1 = lines1[:len(lines2)]
+    elif len(lines1) < len(lines2) and is_nops(lines2[len(lines1):]):
+        lines2 = lines2[:len(lines1)]
```

Comment:
This only trims nops in the zero-score case. We should make it work in other cases too. E.g. based on based on the last chunk of the computed diff, or trimming lines1 and lines2 until they contain equally many trailing nops